src/components/Person/Person.jsx

```diff
@@ -1,27 +1,28 @@
 export const Person = ({ person }) => {
-  const { name, age, sex, isMarried, partnerName } = person;
+  const {
+    name,
+    age,
+    sex,
+    partnerName,
+  } = person;
 
-  const partner = sex === 'm' ? 'wife' : 'husband';
+  const spouse = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  const marriage = person.isMarried
+    ? `${partnerName} is my ${spouse}`
+    : `I am not married`;
 
   return (
     <section className="Person">
-      <h2 className="Person__name">
-        My name is
-        {' '}
-        {name}
-      </h2>
-      {age && (
-        <p className="Person__age">
-          I am
-          {' '}
-          {age}
-        </p>
-      )}
-      {isMarried ? (
-        <p className="Person__partner">{partnerName ? `${partnerName} is my ${partner}` : `I am not married`}</p>
-      ) : (
-        <p className="Person__partner">I am not married</p>
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {person?.age && (
```

Comment:
Check for `person.age >= 0` or `!!person.age` or `person?.age !== undefined` as this can result in outputing '0' to the template as @JoyCoffeeAddict stated.

Reply:
it must be a boolean value